integration_test/Dialect/ESI/runtime/loopback.mlir

```diff
@@ -95,6 +96,7 @@ hw.module @CallableFunc1() {
 }
 
 esi.manifest.sym @Loopback name "LoopbackIP" version "v0.0" summary "IP which simply echos bytes" {foo=1}
+esi.manifest.consts @Loopback {depth=5:ui32}
```

Comment:
I've been bashed for excessive succinctness before, so i'm here going to push back on calling this `consts` instead of `constants` :).